lib/mix/tasks/token.ex

```diff
@@ -40,11 +40,16 @@ defmodule Mix.Tasks.Cadet.Token do
       user = test_user(role)
 
       {:ok, access_token, _} =
-        Guardian.encode_and_sign(user, %{}, token_type: "access", ttl: {4, :weeks})
+        Guardian.encode_and_sign(user, %{}, token_type: "access", ttl: {1, :day})
```

Comment:
Not sure what's the point of making this change tbh, outside of testing the auth controller but I guess it doesn't hurt either.